parser.go

```diff
@@ -64,6 +64,7 @@ func NewAuditMessage(nlm *syscall.NetlinkMessage) *AuditMessage {
 
 // Gets the timestamp and audit sequence id from a netlink message
 func parseAuditHeader(msg *syscall.NetlinkMessage) (time string, seq int) {
+	var err error
```

Comment:
How about returning the error as well? If you return an error, you don't have to reset `time` and `seq` as the consumer can check the error value.